agent-jenkins-tools/Dockerfile

```diff
@@ -3,42 +3,29 @@ FROM quay.io/feedhenry/jenkins-agent-java:latest
 USER root
 
 # Jenkins Job Builder
-
 ENV BRANCH master
 
 WORKDIR /
-RUN git clone -b allow_self_signed_certs https://github.com/mikenairn/python-jenkins.git && \
-    pushd python-jenkins && \
-    yum -y --enablerepo=extras install epel-release && \
+RUN yum -y --enablerepo=extras install epel-release && \
     yum -y install python-pip skopeo && \
     yum clean all && \
     rm -rf /var/cache/yum && \
-    pip install -U -r requirements.txt && python setup.py install && \
-    popd && \
 
     git clone -b ${BRANCH} https://git.openstack.org/openstack-infra/jenkins-job-builder && \
     pushd jenkins-job-builder && \
-    git checkout baca7d15504a93c9d92f15e034470cbdc86f2615 && \
-    /usr/bin/pip install -U pytest && \
-    pip install -U -r requirements.txt && \
-    python setup.py install && \
+        git checkout ${BRANCH} && \
+        /usr/bin/pip install -U pytest && \
+        pip install -U -r requirements.txt && \
+        python setup.py install && \
     popd && \
 
```

Comment:
Having spaces in between the RUN command seems to cause a warning message when building the image:
```
[WARNING]: Empty continuation line found in:
    RUN yum -y --enablerepo=extras install epel-release &&     yum -y install python-pip skopeo &&     yum clean all &&     rm -rf /var/cache/yum &&     git clone -b ${BRANCH} https://git.openstack.org/openstack-infra/jenkins-job-builder &&     pushd jenkins-job-builder &&         git checkout ${BRANCH} &&         /usr/bin/pip install -U pytest &&         pip install -U -r requirements.txt &&         python setup.py install &&     popd &&     git clone https://github.com/feedhenry/jenkins-plugin-install.git &&     pushd jenkins-plugin-install &&         pip install -r requirements.txt && python setup.py install &&     popd
[WARNING]: Empty continuation lines will become errors in a future release.
```
Removing the spaces resolves the issue

Reply:
I must have an earlier version of Docker, since I don't see those warnings. I'll remove them so that it doesn't fail to build in future because of this.

Stupid docker :man_facepalming:

Reply:
@grdryn To be fair I only recently updated docker. Running version 18.03.1